WinUIGallery/AllControlsPage.xaml

```diff
@@ -14,100 +14,71 @@
     xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
     xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
     xmlns:local="using:AppUIBasics"
-    xmlns:behavior="using:AppUIBasics.Behaviors"
-    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
-    NavigationCacheMode="Enabled"
-    HighContrastAdjustment="None">
+    HighContrastAdjustment="None"
+    NavigationCacheMode="Enabled">
 
     <!--  This grid acts as a root panel for the page.  -->
-    <Grid Background="{ThemeResource HomePageBackgroundBrush}">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+
+        <Grid
+            x:Name="HeaderGrid"
+            Height="200"
+            CornerRadius="8,0,0,0">
+            <Image
+                x:Name="headerImage"
+                HorizontalAlignment="Center"
+                Source="Assets/MainPage_Banner.png"
+                Stretch="UniformToFill" />
+            <TextBlock AutomationProperties.AutomationId="__ClickableAreaTextBlock" />
+            <TextBlock
+                x:Name="smallHeaderText"
+                Margin="36,0,0,0"
+                VerticalAlignment="Center"
+                Foreground="Black"
+                Style="{StaticResource TitleTextBlockStyle}"
+                Text="{StaticResource ControlsName}" />
+        </Grid>
+        <GridView
+            x:Name="itemGridView"
+            Grid.Row="1"
+            Padding="24,36,24,36"
+            AutomationProperties.AutomationId="ItemGridView"
+            AutomationProperties.Name="Items In Group"
+            ContainerContentChanging="OnItemGridViewContainerContentChanging"
+            IsItemClickEnabled="True"
+            IsSwipeEnabled="False"
+            ItemClick="OnItemGridViewItemClick"
+            ItemContainerStyle="{StaticResource IndentedGridViewItemStyle}"
+            ItemTemplate="{StaticResource ControlItemTemplate}"
+            ItemsSource="{x:Bind Items, Mode=OneWay}"
+            KeyDown="OnItemGridViewKeyDown"
+            Loaded="OnItemGridViewLoaded"
+            SelectionMode="None" />
         <VisualStateManager.VisualStateGroups>
```

Comment:
I would prefer having VisualState stuff before the actual content of the control/page. This is also how its done everywhere in the framework.

Reply:
Ah yes.. XAML Styler messed up. Fixed!

Reply:
Maybe this got reverted somehow?